Makefile

```diff
@@ -14,60 +14,34 @@
 
 EXTENSION_PREFIX            := gardener-extension
 NAME                        := runtime-gvisor
-REGISTRY                    := eu.gcr.io/gardener-project
+NAME_INSTALLATION           := runtime-gvisor-installation
+CMD_DIRECTORY				:= ./cmd/$(EXTENSION_PREFIX)-$(NAME)
```

Comment:
identation is still somewhat broken here, but well..

Reply:
Somehow the IDE shows with the incorrect  indentation - well worked with vi :)